setup.py

```diff
@@ -30,6 +28,6 @@
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
     ],
-    version='1.4',
+    version='1.5',
```

Comment:
Overall, PR looks good.
The rename of the `use_master_database` decorator warrants a major version bump. Let's make it `2.0`

Reply:
oh you are right, it worth a 2.0